### PR TITLE
[17.0][MIG] stock_picking_purchase_order_link: Migration to 17.0

### PR DESCRIPTION
**Pull request by ramibha**

### CODE REVIEW

File: stock_picking_purchase_order_link/views/stock_picking_view.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <field name="purchase_id" invisible="1" />
+                <button
+                    name="action_view_purchase_order"
+                    class="oe_stat_button"
+                    icon="fa-shopping-cart"
+                    string="Purchase Order"
+                    type="object"
+                    invisible="purchase_id == False"
```
Comment (astirpe) [inline stock_picking_purchase_order_link/views/stock_picking_view.xml]:
```suggestion
                    invisible="not purchase_id"
```

**Review by astirpe — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

@Ramibha Maybe a commit message more like [IMP] stock_picking_purchase_order_link: (...) could be better. Thanks

**Pull request merged**